### PR TITLE
Improve Demo Website

### PR DESCRIPTION
**Pull request by MichaelDeng03**
- Update formatting and style of demo webpage 
- Make new main webpage
- Moved location of webpage servers
- Renamed dockerfiles/docker compose services
- Update make commands to replace localhost_demo... and add aws ecr push commands
- Add asynch streaming to demo website
- Move CLIClient emits to agent.qna + update output formatting
- Switch webpage listener to use 443 listener
- Prompt improvements

Thread updates:
- Remove thread arg from qna
- Make a new thread for each qna, explicitly passing in necessary context + prompt

### CODE REVIEW

File: concrete/orchestrator.py [hunk 1]
```diff
@@ -162,19 +161,27 @@ def communicative_dehallucination(
 
     # Iterative Q&A process
     q_and_a = []
-    for i in range(max_iter):
+    for _ in range(max_iter):
         question = developer.ask_question(context)
-        CLIClient.emit(f"Developer's question:\n {question}\n")
+        # CLIClient.emit(f"Developer's question:\n {question}\n")
```
Comment (abjjabjj) [inline concrete/orchestrator.py:166]: we can delete these

**Comment by MichaelDeng03** [comment]
What's best practice for when you have PR up, but want to continue making changes on that branch

**Comment by MichaelDeng03** [comment]
Should I take the PR down

**Comment by MichaelDeng03** [comment]
push mid-PR, wait for merge, then put up another PR?

**Comment by abjjabjj** [comment]
if the changes are doing something else then it should be another PR

**Comment by abjjabjj** [comment]
even when they are related if you can scope it down into another PR it's usually better

**Comment by abjjabjj** [comment]
nobody wants to review a massive PR

**Comment by abjjabjj** [comment]
i think it's also best practice for there to be one PR per branch

**Comment by abjjabjj** [comment]
Just need to resolve conflicts now

**Comment by MichaelDeng03** [comment]
Should I pull main into my branch and resolve conflicts?

**Comment by abjjabjj** [comment]
Yeah

**Review by abjjabjj — APPROVED** [review]

**Pull request merged**